cvat/apps/git/git.py

```diff
@@ -79,8 +79,8 @@ def _parse_url(self):
             # https://github.com/git/git/blob/77bd3ea9f54f1584147b594abc04c26ca516d987/url.c
 
             host_pattern = r"((?:(?:(?:\d{1,3}\.){3}\d{1,3})|(?:[a-zA-Z0-9._-]+.[a-zA-Z]+))(?::\d+)?)"
-            http_pattern = r"(?:http[s]?://)?" + host_pattern + r"((?:/[a-zA-Z0-9._-]+){2})"
-            ssh_pattern = r"([a-zA-Z0-9._-]+)@" + host_pattern + r":([a-zA-Z0-9._-]+)/([a-zA-Z0-9._-]+)"
+            http_pattern = r"(?:http[s]?://)?" + host_pattern + r"((?:/[a-zA-Z0-9._-]+){2,})"
+            ssh_pattern = r"([a-zA-Z0-9._-]+)@" + host_pattern + r":([a-zA-Z0-9._-]+)((?:/(?:[a-zA-Z0-9._-]+))*)/([a-zA-Z0-9._-]+)"
```

Comment:
I think, in the added fragment, the most internal braces can be omitted: `/(?:[a-zA-Z0-9._-]+)` -> `/[a-zA-Z0-9._-]+`

Why introduce a new group? Do you have an idea how can this can be used in future?

Thanks for the contribution!

Reply:
You are right those braces can be omitted.

Also I reduced complexity of this regex and dropped the additional group. I do not have justification for introduction a new group.